### ISSUE TITLE
generate-commit-list needs to work off tag and/or HEAD

### ISSUE THREAD

**Issue by straker** [dequelabs/axe-api-team-public#91]
After the `release` branch has been merged into `main`/`master`, we want to use [label-and-move-released-issues-v1](https://github.com/dequelabs/axe-api-team-public/tree/main/.github/actions/label-and-move-released-issues-v1) on all commits in the release. However using `generate-commit-list` requires two branches, but once master is merged all branches are in-sync there won't be any differences between the branch commits.

To address this we'll need `generate-commit-list` to accept a tag and the term `HEAD` in order to get a commit list from the last git tag to the current `main`/`master` HEAD.

```
git log v4.8.2..HEAD --oneline --no-merges --abbrev-commit
```
